USAGE.md

```diff
@@ -115,33 +115,51 @@ If `citation-tags.tsv` defines the tag `study-x`, then this study can be cited l
 
 ## Reference metadata
 
-The Manubot workflow requires the bibliographic details for references (the set of all cited works) as CSL (Citation Style Language) Items (also known as [citeproc JSON](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#csl-json-items)).
-The Manubot attempts to automatically retrieve metadata and generate valid citeproc JSON for references, which is exported to `output/references.json`.
-However, in some cases the Manubot fails to retrieve metadata or generates incorrect or incomplete citeproc metadata.
+Manubot stores the bibliographic details for references (the set of all cited works) as CSL JSON ([Citation Style Language Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#csl-json-items)).
+For all citation sources besides `raw`, Manubot automatically generates CSL JSON.
+In some cases, automatic metadata retrieval fails or provides incorrect or incomplete information.
 Errors are most common for `url` references.
-For these references, you can manually specify the correct CSL Data in [`content/manual-references.json`](content/manual-references.json), which will override the automatically generated reference data.
-To do so, create a new CSL JSON Item that contains the field `"standard_citation"` with the appropriate reference identifier as its value.
-The identifier can be obtained from the `standard_citation` column of `citations.tsv`, which is located in the `output` branch or in the `output` subdirectory of local builds.
-As an example, `manual-references.json` contains:
+Therefore, Manubot supports user-provided metadata, which we refer to as "manual references".
+When a manual reference is provided, Manubot uses the supplied metadata and does not attempt to generate it.
+
+Manubot searches the `content` directory for files that match the glob pattern `manual-references*.*` and expects that these files contain manual references.
+[`content/manual-references.json`](content/manual-references.json) is the default file to specify custom CSL JSON metadata.
+Manual references are matched to citations using their "id" field.
+For example, to manually specify the metadata for the citation `@url:https://github.com/manubot/rootstock`, add a CSL JSON Item to `manual-references.json` that contains:
 
 ```json
-"standard_citation": "url:https://github.com/greenelab/manubot-rootstock"
+"id": "url:https://github.com/manubot/rootstock",
```

Comment:
Should we add the `type` here to be consistent with the instructions later?  Maybe we want to denote that this is only an excerpt from the full reference.